site/content/docs/5.2/getting-started/webpack.md

```diff
@@ -1,112 +1,220 @@
+   npm i --save bootstrap @popperjs/core
+   ```
 
-## Importing Styles
+4. **Install additional dependencies.** In addition to Webpack and Bootstrap, we need a few more dependencies to properly import and bundle Bootstrap's CSS and JS with Webpack. These include Sass, some loaders, and Autoprefixer.
 
-### Importing Precompiled Sass
+   ```sh
+   npm i --save-dev autoprefixer css-loader export-loader postcss-loader sass sass-loader style-loader
+   ```
 
-To enjoy the full potential of Bootstrap and customize it to your needs, use the source files as a part of your project's bundling process.
+Now that we have all the necessary dependencies installed and setup, we can get to work creating the project files and importing Bootstrap.
 
-First, create your own `_custom.scss` and use it to override the [built-in custom variables]({{< docsref "/customize/sass" >}}). Then, use your main Sass file to import your custom variables, followed by Bootstrap:
+## Project structure
 
-```scss
-@import "custom";
-@import "~bootstrap/scss/bootstrap";
-```
+We've already created the `my-project` folder and initialized npm. Now we'll also create our `src` and `dist` folders to round out the project structure. Run the following from `my-project`, or manually create the folder and file structure shown below.
 
-For Bootstrap to compile, make sure you install and use the required loaders: [sass-loader](https://github.com/webpack-contrib/sass-loader), [postcss-loader](https://github.com/webpack-contrib/postcss-loader) with [Autoprefixer](https://github.com/postcss/autoprefixer#webpack). With minimal setup, your webpack config should include this rule or similar:
-
-<!-- eslint-skip -->
-```js
-// ...
-{
-  test: /\.(scss)$/,
-  use: [{
-    // inject CSS to page
-    loader: 'style-loader'
-  }, {
-    // translates CSS into CommonJS modules
-    loader: 'css-loader'
-  }, {
-    // Run postcss actions
-    loader: 'postcss-loader',
-    options: {
-      // `postcssOptions` is needed for postcss 8.x;
-      // if you use postcss 7.x skip the key
-      postcssOptions: {
-        // postcss plugins, can be exported to postcss.config.js
-        plugins: () => {
-          [
-            require('autoprefixer')
-          ];
-        }
-      }
-    }
-  }, {
-    // compiles Sass to CSS
-    loader: 'sass-loader'
-  }]
-}
-// ...
+```sh
+mkdir {dist,src,src/js,src/scss}
+touch dist/index.html src/js/main.js src/scss/styles.scss src/scss/_custom.scss
 ```
 
-### Importing Compiled CSS
-
-Alternatively, you may use Bootstrap's ready-to-use CSS by simply adding this line to your project's entry point:
-
-<!-- eslint-skip -->
-```js
-import 'bootstrap/dist/css/bootstrap.min.css';
+When you're done, your complete project should look like this:
+
+```text
+my-project/
+├── dist/
+│   └── index.html
+├── src/
+│   ├── js/
+│   │   └── main.js
+│   └── scss/
+│       ├── _custom.scss
+│       └── styles.scss
+├── package-lock.json
+├── package.json
+└── webpack.config.js
```

Comment:
I'd remove this line since this file doesn't exist at this point. When you follow the step and you don't read the next sentence the user could think I missed some steps.
If this line is removed maybe change the next paragraph to include the creation of this file or in the next part something like "Create and open webpack.config.js in your editor".

Reply:
or put 
_"At this point, everything is in the right place, but Webpack won't work because we haven't created our `webpack.config.js` file yet."_
above the graph?

Reply:
Modified the text here to clarify this all.